x-pack/legacy/plugins/apm/public/components/app/Settings/SettingsList.tsx

```diff
@@ -173,6 +173,25 @@ export function SettingsList() {
     />
   );
 
+  const failurePrompt = (
+    <EuiEmptyPrompt
+      iconType="alert"
+      body={
+        <>
+          <p>
+            {i18n.translate(
+              'xpack.apm.settings.agentConf.configTable.failurePromptText',
+              {
+                defaultMessage:
+                  'The list of agent configurations could not be fetched'
```

Comment:
I know I'm going back and forth on this but my assumptions have changed a little.
If we do not expect the `apm_user` to have read access, and we therefore expect many users to see this issue because of permission problems, it might make sense to hint about that.

Eg. "The list of agent configurations could not be fetched. Your user may not have the sufficient permissions."

To be clear, I was more against the fact that the user was seeing an error for something they shouldn't be concerned about (they were seeing an issue if they didn't have write access, even if they just wanted to view/read the list).